_posts/2019-01-09-tools-we-use.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Tools We Use at Ombu Labs"
+date: 2019-01-09 10:00:00
+categories: ["agile"]
+author: "emily"
+---
+
+At Ombu Labs, we have a number of tools that help us to keep organized, write better code and communicate more effectively with our clients.
+
+<!--more-->
+
+### Daily Work
+- We use [Slack](https://ombulabs.slack.com/) to chat with team members while working.
+- We use [Trello](http://trello.com) to manage our clients and leads.
+- We use [Freckle](https://letsfreckle.com/) to track our time.
+
+### Code
+- We use [Atom](https://atom.io/) or [Sublime Text](http://www.sublimetext.com/) to write code.
```

Comment:
@ercohen14 I think @rdormer uses `vim` as his preferred editor. To be honest we don't have a default editor or a recommended editor. I think everyone should use whatever they're most comfortable with... In the past I've used many other editors: MacVim, Sublime, Aptana (based on Eclipse), and I'm currently using Atom. As a team member you should feel free to use whatever you want. 😃